CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to TEAMMATES
+If you are just starting out, do refer to the [Contributor's Orientation Guide][COG].  
+Also, remember to follow the [TEAMMATES Development Process][dev].
+
+[COG]: https://docs.google.com/document/d/1cY9pmEmw4dC6Z4LjN1WUTsynJ0jVPgbWsFhdyBy_wCU/pub?embedded=true
+[dev]: /devdocs/process.md
+
+### Other Resources
+For a comprehensive directory of resources, including **style guides** and **best practices**, please refer to [README.md][readme].
```

Comment:
This section can be merged with the above (under "Submitting a patch") because it only concerns anyone who wants to contribute patches anyway.